doc/contributing/hw/racl/README.md

```diff
@@ -71,26 +99,9 @@ Source processing elements may have a static role assignment or can support a dy
         * Note: It is not okay to hang by not returning a completion.
         * A design parameter dictates if a bus error is returned or not.
    - If the RACL was violated
-      + If a previously logged error is present then set the overflow bit else Log an error in an error logging register
-
-5. Error-logging register (8-bit) - used to log errors due to RACL violation.
-   This register is readable and writable by ROT role and so is governed by the special `ROT_PRIVATE` policy.
-   This register should be CTN mapped.
-   This register is primarily intended as a debug aid and is not intended to cause any kind of alerts.
-   + Bit 6 - `ErrorValid`
-     * Set to 1 if a valid error is logged in the register.
-   + Bit 5 - `Overflow`
-     * A second error occurred when ErrorValid was 1
-   + Bit 4 - `Read-or-Write`
-     * Set to 0 if a read request violated the RACL
-     * Set to 1 if a write/read-modify-write that violated the RACL
-   + Bits 3:0 - Role that violated the RACL
-
-## Fabric Requirements
-The fabric has a 4-bit field defined in the transaction to carry the role that originated the transaction request such that the ACL rules can be enforced based on the role.
-In the case of TLUL, the a_user bits 21 through 18 are used to carry this information.
-
-## Processing Element Requirements
+      + If a previously logged error is present then set the overflow bit else log an error in an error logging register
```

Comment:
This also applies to the previous version of the text, but I think it might be better to write it the other way around:

```
Update the RACL error log (of type `racl_error_log_t`) to reflect the reported error, setting the `valid` field.
If the log was not already valid, report the error by setting the `racl_role`, `ctn_uid`, `read_access` and `request_address` fields.
If it was valid, leave those fields unchanged, but set the `overflow` field.
```